samples_project/Assets/SampleViewer/Samples/Measure/Measure.cs

```diff
@@ -112,15 +114,19 @@ private void Interpolate(GameObject start, GameObject end, List<GameObject> feat
 
             // Calculate transform of next point.
             float nextX = pre.x + (float)dx;
+            float nextY = pre.y + (float)dy;
             float nextZ = pre.z + (float)dz;
-            next.transform.position = new Vector3(nextX, 0, nextZ);
+            next.transform.position = new Vector3(nextX, nextY, nextZ);
 
             // Set default location component of next point.
             next.GetComponent<ArcGISLocationComponent>().enabled = true;
             next.GetComponent<ArcGISLocationComponent>().Rotation = new ArcGISRotation(0, 90, 0);
 
-            // Define height.
-            SetElevation(next);
+            // Obsolete: would raycast elevation to always be on groud level, now just match to the line.
+            //SetElevation(next);
+            var location = next.GetComponent<ArcGISLocationComponent>();
```

Comment:
Can you give the name `next` something more descriptive, like `nextPosition` or something?